website/blog/2023-01-04-ferretdb-beta-version-authentication.md

```diff
@@ -0,0 +1,117 @@
+
+4. Connect to PostgreSQL and drop `ferredb` database as it's not needed anymore:
+   * `psql -h localhost -p 5432 -U username postgres`
+   * `DROP DATABASE ferretdb`
+
+5. Upgrade FerretDB and run FerretDB 0.8 (Please refer to our documentation where we describe how to update and start FerretDB)
+
+6. Restore database using `mongorestore --uri="mongodb://localhost:27017"`
+
+7. In case something doesn't work and you need to rollback to FerretDB 0.7.1:
+   * Stop ferretdb
+   * Delete ferretdb database (repeat step 4)
+   * Restore `ferretdb` PostgreSQL db from the dump we created on the step 2:  `psql -h localhost -p 5432 -U username ferretdb -f ferretdb.sql`
+   * Start ferretdb 0.7.1
+:::
+
+In this blog post, we'll be sharing detailed information about the FerretDB beta release (0.8.0).
+
+## What's new?
+
+Since the last release (0.7.2), we've made great strides in adding new features to FerretDB, especially the introduction of authentication for PostgreSQL.
+With authentication, users can now connect securely using passwords, ensuring that only authorized connections are established.
+You can implement an authenticated connection by specifying the connection string in this format: `postgres://username:password@postgres:5432/ferretdb`.
+See more details [in our documentation](https://docs.ferretdb.io/security/#authentication).
+
+But that's not all - in addition to the `$max` update operator, FerretDB beta now includes support for the `$min` update operator:
+
+```js
+db.collection.update( {}, { $min: { <field1>: <value1>, ... } } )
+```
+
+We've also added support for `ordered` inserts, which allows you to insert data in the exact order it comes in.
+
+```js
+db.collection.insert( { "field1": "value1" }, { ordered: true } )
+```
+
+## Bug fixes and enhancements
+
+This release includes a bug fix that addresses an issue with the `$inc` operator when used with `unset` documents.
+We found that, in certain cases, an invalid value of the `$inc` operator was updating `unset` documents with non-numeric values when it should have instead returned an error.
+This release should prevent unexpected updates on `unset` documents, and likewise, improve the overall stability and reliability of FerretDB.
+
+The release includes one enhancement, which is updating our building documentation.
+We have documented our build process and Go build tags for those who use embedded packages or build FerretDB themselves.
+Please [check here](https://github.com/FerretDB/FerretDB/blob/main/README.md#building-and-packaging) for further details.
+
+## Documentation
+
+Our documentation is not left out on the list of improvements!
+Specifically, we've updated it to include descriptions and examples for element query operators, array query operators, and comparison and logical query operators.
+
+These changes help make our documentation more comprehensive and user-friendly, which should hugely benefit our community and users alike.
```

Comment:
Maybe in such sentences, we can ask for feedback? Like "don't hesitate to let us know what you think about it" or something :)